### PR TITLE
bump kic base image

### PR DESCRIPTION
**Pull request by medyagh**
Pushed new kic image to include https://github.com/kubernetes/minikube/pull/9029

This PR should be converted to an Auto PR by minikube-bot

### CODE REVIEW

File: pkg/drivers/kic/types.go [hunk 1]
```diff
@@ -24,9 +24,9 @@ import (
 
 const (
 	// Version is the current version of kic
-	Version = "v0.0.12-snapshot"
+	Version = "v0.0.12-snapshot2"
```
Comment (tstromberg) [inline pkg/drivers/kic/types.go:27]: What's with the snapshot name? Why not versions?

Comment (medyagh) [reply pkg/drivers/kic/types.go:27]: This is before we release.

Comment (medyagh) [reply pkg/drivers/kic/types.go:27]: so basically snapshot means current HEAD changes that has not been released to people yet
we can potentially use a different repo too but this pattern seems to straighforward

Comment (medyagh) [reply pkg/drivers/kic/types.go:27]: the idea is to one day automate this process using this pattern https://github.com/kubernetes/minikube/issues/9072

Comment (afbjorklund) [reply pkg/drivers/kic/types.go:27]: Probably need some additional identifier too, to distinguish between different images.
Could either use commit (from "git describe") or a date (like KIND is doing) perhaps ?

v1.12.3-153-g5dc6c83
v20200817-222e7545

**Pull request merged**